src/schema/v2/__tests__/reprocessArtworkImageMutation.test.ts

```diff
@@ -4,7 +4,9 @@ import gql from "lib/gql"
 describe("ReprocessArtworkImageMutation", () => {
   const mutation = gql`
     mutation {
-      reprocessArtworkImage(input: { artworkID: "artwork-id", imageID: "image-id" }) {
```

Comment:
👀

Reply:
Yeah – prettier _really_ wanted to reformat this line 😅